src/sourmash/cli/info.py

```diff
@@ -1,7 +1,6 @@
 """display sourmash version and other information"""
 
 import os
-import screed
```

Comment:
moving `screed` from the top-level to inside functions were it is used to avoid import errors in pyodide. Will be fixed by https://github.com/dib-lab/screed/issues/98